hdw-a339x/src/systems/tcas/src/components/TcasComputer.ts

```diff
@@ -367,14 +379,12 @@ export class TcasComputer implements TcasComponent {
             }),
         ),
       )
-        .then((_unused) => {
-          this.vatsimDataThrottler = setTimeout(() => this.updateVatsimData(), 1000 * 5);
-        })
-        .catch((_unused) => {
-          this.vatsimDataThrottler = setTimeout(() => this.updateVatsimData(), 1000 * 5);
-        });
-    } else {
-      this.vatsimDataThrottler = setTimeout(() => this.updateVatsimData(), 1000 * 5);
+      .then((_unused) => {
+        // _unused
```

Comment:
Actually the timing reset needs to happen here, because otherwise there might be a second update initiated while the first is still running(in the case the update takes more then 5 secs).
Something like
```js
 private updateTrafficData(_deltaTime: number): void {
    if (this.selectedTrafficDataSource === 0 || this.secondsSinceLastTrafficDataUpdate === -1) {
      return;
    }

...
    this.secondsSinceLastTrafficDataUpdate = -1;
...
.then((_unused) => {
           this.secondsSinceLastTrafficDataUpdate = 0;
      })
      .catch((_unused) => {
            this.secondsSinceLastTrafficDataUpdate = 0;
      });
```